content/applications/hr/lunch/vendors.rst

```diff
@@ -0,0 +1,152 @@
+=======
+Vendors
+=======
+
+Before :doc:`products can be added <products>` to the *Lunch* app, the restaurants that provide the
+food **must** be configured.
+
+To add a new vendor, first navigate to :menuselection:`Lunch app --> Configuration: Vendors`. Here,
```

Comment:
```suggestion
To add a new vendor, first navigate to :menuselection:`Lunch app --> Configuration --> Vendors`. Here,
```
Remember: we only use ":" formatting when it has to do with an option that part of a menu's sub section... This is just an option available under the configuration menu -- without being a part of any sub section

Reply:
I swear I "know" that but I just crossed my wires here =D